packages/web/src/components/gcds-textarea/gcds-textarea.stories.tsx

```diff
@@ -63,41 +63,41 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-textarea
   textarea-id="${args.textareaId}"
   label="${args.label}"
```

Comment:
Same comment about the label: would you mind marking it as required in the story and setting a default data type?